### DISCUSSION TITLE
DIY 20A SCT006?

### DISCUSSION THREAD

**Discussion by pyrodex** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#168] [Q&A]
So, I need to get my hands on some more SCT006 20A units and can find them on AliExpress but they aren't terminated with 3.5MM jacks. Since these have built in TVS just without terminations could I simply buy a MONO 3.5MM cable and do it myself? 

Would this work?
![image](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/assets/108703/bd6874f4-552b-46a9-8365-ca39a6192cc1)

I don't know which end goes to which on the pigtail from the SCT006. 

Thanks!

**Comment by CircuitSetup** [comment]
It doesn't really matter which wire goes to which, unless you insist that the arrow on the CT is pointing the right way. If you hook it up and get a negative value in the output, then turn the CT around on the wire.
 
We have them in stock at CircuitSetup too :) (it may say they're out of stock at the moment)

**Reply by pyrodex** [reply]
So, I don't need any TVS or any other components in line other than a simple MONO speaker cable?

**Reply by CircuitSetup** [reply]
You shouldn't - the ones with a mA output have the TVS built in. I don't know that I've seen them with voltage output anyway (which would have a burden resistor built in)

**Reply by TRSx80** [reply]
> We have them in stock at CircuitSetup too :) (it may say they're out of stock at the moment)

Is this still the case (I see you replied back in January, but they still show as out of stock)?

**Comment by TRSx80** [comment]
It would be lovely if you could share the proper orientation once you figure it out, for us OCD folks out here who may need to do the same thing some time!

It's not necessarily theoretical either, web shop seems to be out of the small CT clamps at the moment.

**Reply by pyrodex** [reply]
For me it was white to white and red to red. Orientation of the ct clamp in the panel is always fun so I have to flip them sometimes to ensure I am getting a positive value of course.
